src/main/kotlin/mate/academy/model/Figure.kt

```diff
@@ -0,0 +1,7 @@
+package mate.academy.model
+
+import mate.academy.properties.Area
+import mate.academy.properties.Draw
+
+sealed class Figure(val color: Color): Area, Draw {
```

Comment:
Using sealed class this is not standard practice for this purpose.
Please consider using of abstract class